charts/stable/gollum/templates/_pod.yaml

```diff
@@ -0,0 +1,78 @@
+{{- /*
+The pod definition included in the controller.
+*/ -}}
+{{- define "common.controller.pod" -}}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- toYaml . | nindent 2 }}
+{{- end }}
+serviceAccountName: {{ include "common.names.serviceAccountName" . }}
+{{- with .Values.podSecurityContext }}
+securityContext:
+  {{- toYaml . | nindent 2 }}
+{{- end }}
+{{- with .Values.priorityClassName }}
+priorityClassName: {{ . }}
+{{- end }}
+{{- with .Values.schedulerName }}
+schedulerName: {{ . }}
+{{- end }}
+{{- with .Values.hostNetwork }}
+hostNetwork: {{ . }}
+{{- end }}
+{{- with .Values.hostname }}
+hostname: {{ . }}
+{{- end }}
+{{- if .Values.dnsPolicy }}
+dnsPolicy: {{ .Values.dnsPolicy }}
+{{- else if .Values.hostNetwork }}
+dnsPolicy: "ClusterFirstWithHostNet"
+{{- else }}
+dnsPolicy: ClusterFirst
+{{- end }}
+{{- with .Values.dnsConfig }}
+dnsConfig:
+  {{- toYaml . | nindent 2 }}
+{{- end }}
+enableServiceLinks: {{ .Values.enableServiceLinks }}
+initContainers:
+{{- with .Values.initContainers }}
+  {{- toYaml . | nindent 2 }}
+{{- end }}
+  - name: git-init
+    image: alpine/git
+    args:
+      - clone 
+      - -b
+      - {{ .Values.init.gitBranch  }}
+      - {{ .Values.init.gitUrl  }}
+      - /wiki
+    volumeMounts:
+      - mountPath: /wiki
+        name: wiki
```

Comment:
All init containers can also be defined without redefining the common.controller.pod template. This can be done in a common.yaml as well and injected.